client/src/components/Home/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import NavBar from "../Navbar";
+import "./style.css";
+import Category from "../Category";
+class Home extends React.Component {
+  goToShop = () => {
+    const { history } = this.props;
+    history.push("/shop");
+  };
+  render() {
+    return (
+      <div className="home">
+        <NavBar />
```

Comment:
Since this component has no state transfer to a functional component

Reply:
i change it !